README.md

```diff
@@ -43,6 +43,7 @@
 
 * [Protofire](http://protofire.io) - Protofire specializes in software engineering for decentralized infrastructures, protocols and applications.
 * [ScopeLift](https://www.scopelift.co/) - Software consultancy with expertise in crypto, web, and native mobile development.
+* [4IRElabs](https://4irelabs.com/) – research and development for DeFi, DAOs and more
```

Comment:
Thanks @MaxSemenchuk! This is great. Do you mind moving it to the top of the list, since I generally consider digits before alphabet characters alphabetically. Also, if you don't mind, capitalize the "R" in research. Thanks :)